fe/fe-core/src/main/java/org/apache/doris/analysis/StmtRewriter.java

```diff
@@ -1141,5 +1147,32 @@ private static Expr createJoinConjunct(Expr exprWithSubquery, InlineViewRef inli
         smap.put(subquery, subquerySubstitute);
         return exprWithSubquery.substitute(smap, analyzer, false);
     }
+
+    @SneakyThrows
+    public static void rewriteByPolicy(SelectStmt selectStmt, Analyzer analyzer) {
+        Catalog currentCatalog = Catalog.getCurrentCatalog();
+        for (int i = 0; i < selectStmt.fromClause_.size(); i++) {
```

Comment:
The `TableRef` is not only in `from` clause.
You need to traverse the top level `selectStmt` to find all of them. maybe in subquery.

Reply:
Ok, thanks for the reminder, I'll fix it